### DISCUSSION TITLE
rejection of student benefits

### DISCUSSION THREAD

**Discussion by ManojKumarMithireddy** [community/community#77908] [GitHub Education]
### Hi everyone,

I was trying to get student benefits in github . without any prior knowlwdge I started submitting my application ,then I faced an issue to change the name in the profile to same as in the proof. After setting my name I then again applied an application but it is showing me "Your uploaded proof has already been used previously." as error message .
yeah I used it previously but I didn't get my application approved 
please solve this issue
![image](https://github.com/community/community/assets/148680918/00c3cc28-ca58-46a4-bbdb-a86e990c3f17)

**Comment by ManojKumarMithireddy** [comment]
yeah the issue is resolved.
I just needed to apply with another photo of my ID

**Reply by maverick9460** [reply]
bro please help me i didn't find any solution for  that

**Reply by ManojKumarMithireddy** [reply]
Bro can you be more specific about your problem

**Reply by maverick9460** [reply]
bro iam facing same problem that u face  that u uploded that screeshot

**Reply by maverick9460** [reply]
![Screenshot 2024-02-27 200331](https://github.com/community/community/assets/160167410/79feea99-2171-4b61-ae4d-7947d5d81a2d)

**Reply by maverick9460** [reply]
that problem 
please help me

**Reply by ManojKumarMithireddy** [reply]
Bro try using another pic of the id.
**Don't use same pic**

**Reply by maverick9460** [reply]
kr ke dekh liya

**Reply by maverick9460** [reply]
fir bhe same problem ho rhi ha  mne kbhi upload bhe nhi kiy ha

**Reply by maverick9460** [reply]
broo kr nh yrr kuvh

**Comment by ManojKumarMithireddy** [comment]
I would like someone to resolve the issue of @maverick9460

**Reply by maverick9460** [reply]
bro kr  yr help

**Reply by ManojKumarMithireddy** [reply]
Bro I too don't know about it that much.
Try again with another picture and see if it works that's all I know
